ironfish-cli/src/commands/chain/genesisblock.ts

```diff
@@ -36,6 +40,12 @@ export default class GenesisBlockCommand extends IronfishCommand {
       default: 'Genesis Block',
       description: 'The memo of the block',
     }),
+    allocations: Flags.string({
+      required: false,
+      description:
+        'A CSV file with the format address,amountInIron,memo containing genesis block allocations',
+      exclusive: ['account', 'memo'],
+    }),
     genesisSupplyInIron: Flags.integer({
```

Comment:
I think this needs to be changed back to a string. I changed it to `number` but didn't think about decimal places

Reply:
Good idea. I updated this